StRoot/StFwdTrackMaker/include/Tracker/ObjExporter.h

```diff
@@ -33,6 +31,7 @@ class ObjExporter {
         int p, s, i, j;
         float x, y, z, out;
         int nPitch = nLongitude + 1;
+        const float DEGS_TO_RAD = 3.14159f/180.0f;
```

Comment:
No need to change this, but, FYI, I tried compiling this class with `DEGS_TO_RAD = TMath::DegToRad()` and it compiled, so somewhere amidst all the included dependencies, `TMath.h` is already included :-) I'm sure the compiler optimizes this calculation away such that it isn't done at run-time.